rust/oak/src/grpc/mod.rs

```diff
@@ -157,8 +153,8 @@ pub fn event_loop<T: OakNode>(
             info!("no pending message; poll again");
             continue;
         }
-        if !handles.is_empty() {
-            panic!("unexpected handles received alongside gRPC response")
+        if handles.is_empty() {
+            panic!("no response handle received alongside gRPC request")
```

Comment:
I take it we are just going to refer to handles by their position in the handles array? i.e. if we wanted to send multiple handles, the recipient would have to know what they mean based on their order. FIDL makes this nicer since essentially gives you ways of referring to handles from the messages. Maybe we could approximate something like that by making handles a map from string ("names") to actual handles? Let's talk about it offline anyways, definitely not something for this PR.